packages/gatsby-plugin-treat/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "gatsby-plugin-treat",
+  "description": "Gatsby plugin for treat",
+  "version": "0.1.0",
+  "keywords": ["gatsby", "gatsby-plugin", "treat"],
+  "repository": "https://github.com/seek-oss/treat/tree/master/packages/gatsby-plugin-treat",
+  "main": "index.js",
+  "dependencies": {
+    "babel-plugin-treat": "^1.1.0",
+    "mini-css-extract-plugin": "^0.8.0",
+    "treat": "^1.0.3"
```

Comment:
`treat` will need to be a peer dependency as consumers need to import it directly.